packages/web/tests/dotlottie.test.ts

```diff
@@ -11,6 +11,7 @@ import { createCanvas, sleep } from './test-utils';
 const wasmUrl = new URL('../src/core/dotlottie-player.wasm', import.meta.url).href;
 const jsonSrc = new URL('./__fixtures__/test.json', import.meta.url).href;
 const src = new URL('./__fixtures__/test.lottie', import.meta.url).href;
+const failingAnimation = new URL('./__fixtures__/failingLottie.lottie', import.meta.url).href;
```

Comment:
Suggestion:
Let’s remove this fixture since we’re already stubbing the dotLottie bindings's `render` method to throw an error in the test